23:14:37,184 AutoGPT INFO Welcome back! : Would you like me to return to being Entrepreneur-GPT?
23:15:02,243 openai DEBUG message='Request to OpenAI API' method=post path=https://api.openai.com/v1/embeddings
23:15:02,243 openai DEBUG api_version=None data='{"input": ["[]"], "model": "text-embedding-ada-002", "encoding_format": "base64"}' message='Post details'
23:16:17,262 urllib3.connectionpool WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x1351d1c30>, 'Connection to api.openai.com timed out. (connect timeout=600)')': /v1/embeddings
23:17:32,282 urllib3.connectionpool WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x1351d1d80>, 'Connection to api.openai.com timed out. (connect timeout=600)')': /v1/embeddings
23:20:15,859 AutoGPT INFO Welcome back! : Would you like me to return to being Entrepreneur-GPT?
23:20:22,600 AutoGPT INFO Welcome to Auto-GPT! : Enter the name of your AI and its role below. Entering nothing will load defaults.
23:20:22,943 AutoGPT INFO Name your AI: : For example, 'Entrepreneur-GPT'
23:20:32,728 AutoGPT INFO 'editor' here!: I am at your service.
23:20:32,853 AutoGPT INFO Describe your AI's role: : For example, 'an AI designed to autonomously develop and run businesses with the sole goal of increasing your net worth.'
23:21:13,60 AutoGPT INFO Enter up to 5 goals for your AI: : For example: 
Increase net worth, Grow Twitter Account, Develop and manage multiple businesses autonomously'
23:22:31,514 openai DEBUG message='Request to OpenAI API' method=post path=https://api.openai.com/v1/embeddings
23:22:31,514 openai DEBUG api_version=None data='{"input": ["[]"], "model": "text-embedding-ada-002", "encoding_format": "base64"}' message='Post details'
23:23:46,531 urllib3.connectionpool WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x15083d6c0>, 'Connection to api.openai.com timed out. (connect timeout=600)')': /v1/embeddings
23:25:01,537 urllib3.connectionpool WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x15083d810>, 'Connection to api.openai.com timed out. (connect timeout=600)')': /v1/embeddings
23:26:55,461 AutoGPT INFO Welcome back! : Would you like me to return to being 'editor'?
23:27:10,392 openai DEBUG message='Request to OpenAI API' method=post path=https://api.openai.com/v1/embeddings
23:27:10,392 openai DEBUG api_version=None data='{"input": ["[]"], "model": "text-embedding-ada-002", "encoding_format": "base64"}' message='Post details'
23:28:25,413 urllib3.connectionpool WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x11dd75270>, 'Connection to api.openai.com timed out. (connect timeout=600)')': /v1/embeddings
23:29:40,452 urllib3.connectionpool WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x11dd753c0>, 'Connection to api.openai.com timed out. (connect timeout=600)')': /v1/embeddings
07:59:57,132 AutoGPT INFO Welcome back! : Would you like me to return to being 'editor'?
08:00:02,372 openai DEBUG message='Request to OpenAI API' method=post path=https://api.openai.com/v1/embeddings
08:00:02,373 openai DEBUG api_version=None data='{"input": ["[]"], "model": "text-embedding-ada-002", "encoding_format": "base64"}' message='Post details'
08:01:17,389 urllib3.connectionpool WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x11e531a80>, 'Connection to api.openai.com timed out. (connect timeout=600)')': /v1/embeddings
08:02:32,693 urllib3.connectionpool WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x11e531bd0>, 'Connection to api.openai.com timed out. (connect timeout=600)')': /v1/embeddings
08:07:07,740 AutoGPT INFO Welcome back! : Would you like me to return to being 'editor'?
08:07:10,139 openai DEBUG message='Request to OpenAI API' method=post path=https://api.openai.com/v1/embeddings
08:07:10,139 openai DEBUG api_version=None data='{"input": ["[]"], "model": "text-embedding-ada-002", "encoding_format": "base64"}' message='Post details'
08:08:25,166 urllib3.connectionpool WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x119c3c040>, 'Connection to api.openai.com timed out. (connect timeout=600)')': /v1/embeddings
08:09:40,179 urllib3.connectionpool WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x119c3c160>, 'Connection to api.openai.com timed out. (connect timeout=600)')': /v1/embeddings
09:43:01,67 AutoGPT INFO Welcome back! : Would you like me to return to being 'editor'?
09:43:03,586 openai DEBUG message='Request to OpenAI API' method=post path=https://api.openai.com/v1/embeddings
09:43:03,586 openai DEBUG api_version=None data='{"input": ["[]"], "model": "text-embedding-ada-002", "encoding_format": "base64"}' message='Post details'
09:44:18,600 urllib3.connectionpool WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x150571a80>, 'Connection to api.openai.com timed out. (connect timeout=600)')': /v1/embeddings
09:45:33,615 urllib3.connectionpool WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x150571bd0>, 'Connection to api.openai.com timed out. (connect timeout=600)')': /v1/embeddings
